src/test/java/uk/gov/hmcts/reform/profilesync/domain/SyncJobAuditTest.java

```diff
@@ -3,32 +3,35 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 import java.time.LocalDateTime;
+
 import org.junit.Test;
+import uk.gov.hmcts.reform.profilesync.constants.Source;
 
-public class SyncJobTest {
+public class SyncJobAuditTest {
 
+    private String success = "success";
+    private String errorMsg = "errorMsg";
```

Comment:
add final keyword

Reply:
this has been addressed in the following commit: https://github.com/hmcts/rd-profile-sync/pull/181/commits/bc37603637e5fcbbc46bdd8621f47415d846e2c2